### DISCUSSION TITLE
Passing output from dflook/terraform-apply@v1 to following step

### DISCUSSION THREAD

**Discussion by larryebaum** [dflook/terraform-github-actions#247] [General]
If I'm understanding the docs correct, if my TF file contains outputs, then an output will be produced with the same value.
```
output "app_publicip" {
  description = "Public IP of the instance"
  value = aws_lightsail_instance.app_instance.public_ip_address
}
```
So with a step:
```
    - name: Terraform Apply
      id: tf-apply
<truncated>
```
I'm thinking a subsequent step could reference:
```
    - name: Print the IP
      run: echo "The IP is ${{ steps.tf-apply.outputs.public_ip_address }}"  
```
Alas, no luck as the subsequent steps don't seem to get the value from the tf-apply step. I'm sure its a 'me' problem, what am I overlooking?

**Comment by dflook** [comment]
It looks like you have the wrong name for the output, it should look like:

```yaml
    - name: Print the IP
      run: echo "The IP is ${{ steps.tf-apply.outputs.app_publicip }}"  
```

**Reply by larryebaum** [reply]
Yep, a 'me' problem... I was accidentally referencing the resource value as opposed to the output 'name'. Easy fix and thanks much for the fast reply!!!
